### PR TITLE
Slick Components

### PR DESCRIPTION
**Pull request by salesfelipe**
#### What is the purpose of this pull request?
Add the component Arrow and Dots to a new npm module called: @vtex/slickComponents

#### What problem is this solving?
As these components are used in other components, is best to create them as an npm module and refact the others.

#### Screenshots or example usage

npm i @vtex/storeComponents

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: packages/slick-components/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# BuyButton
```
Comment (brunojdo) [inline packages/slick-components/README.md]: Modify this readme

File: packages/slick-components/package.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@vtex/slick-components",
+  "version": "1.0.1",
```
Comment (brunojdo) [inline packages/slick-components/package.json]: 1.0.0 or 1.0.1?

File: packages/buy-button/package.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@vtex/buy-button",
-  "version": "0.1.0",
+  "version": "0.1.1",
```
Comment (brunojdo) [inline packages/buy-button/package.json]: update the Changelog

File: packages/slick-components/README.md (hunk 1 above)
Comment (salesfelipe) [reply packages/slick-components/README.md]: 👍

File: packages/slick-components/package.json (hunk 2 above)
Comment (salesfelipe) [reply packages/slick-components/package.json]: Cause everytime that I run the lerna publish it automatically changes the version

Comment (salesfelipe) [reply packages/slick-components/package.json]: 👍

File: packages/buy-button/package.json (hunk 3 above)
Comment (salesfelipe) [reply packages/buy-button/package.json]: 👍

**Pull request merged**